worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/SingleThreadQueueExtent.java

```diff
@@ -186,7 +188,11 @@ public synchronized void init(Extent extent, IChunkCache<IChunkGet> get, IChunkC
             };
         }
         if (set == null) {
-            set = (x, z) -> CharSetBlocks.newInstance();
+            if(Objects.equals(Fawe.platform().getPlatform(), "Forge-Official")) {
+                set = (x, z) -> IntSetBlocks.newInstance();
+            } else {
+                set = (x, z) -> CharSetBlocks.newInstance();
+            }
```

Comment:
I think this should be extracted to a util method somewhere else, e.g. in the `BlocksHelper` class

Reply:
I've extracted it to a new ```QueueHelper``` utility class.